crates/sui-node/src/main.rs

```diff
@@ -136,3 +139,15 @@ async fn main() -> Result<()> {
 
     Ok(())
 }
+
+fn get_git_rev() -> String {
+    let output = Command::new("git")
```

Comment:
@longbowlu I am using this cmd as we chatted last time, can you confirm on this cmd?
or do we want this instead for short hash like below?
```
git rev-parse --short HEAD
f86154327
```
```

Reply:
`git describe --always --dirty` is better as it outputs tags whenever possible

Reply:
SG! anything else I should address?